readme.md

```diff
@@ -1,848 +1,1214 @@
-# Code style guide for [CKEditor](https://github.com/ckeditor/ckeditor-dev)
+# Code style guide for [CKEditor 4](https://ckeditor.com/ckeditor/ckeditor4)
 
-**This document is based on [Idiomatic.JS](https://github.com/rwldrn/idiomatic.js).** However, it contains many clarifications and rules specific for projects maintained by [CKSource](http://cksource.com/) like [CKEditor](https://github.com/ckeditor/ckeditor-dev).
+**Table of contents**
 
+* [2. Code style guide](#2-code-style-guide)
```

Comment:
Starting numeration from `2` looks not natural. I feel it is better to add at least 1. Introduction before it, for not confusing anyone.

Reply:
I will add missing headers instead, so numbering is consistent with headers.